aopy/visualization.py

```diff
@@ -204,12 +216,13 @@ def plot_raster(data, plot_cue, cue_bin, ax):
        Create a raster plot of neural data
 
        Args:
-           data (n_trials, n_neurons, n_timebins): neural spiking data (not spike count- must contain only 0 or 1) in the form of a three dimensional matrix
-           plot_cue : If plot_cue is true, a vertical line showing when this event happens is plotted in the rastor plot
-           cue_bin : time bin at which an event occurs. For example: Go Cue or Leave center
+            data (n_trials, n_neurons, n_timebins): neural spiking data (not spike count- must contain only 0 or 1) in the form of a three dimensional matrix
```

Comment:
Can we make the first dimension time and change n_neurons to n_channels? If using multiunit activity we don't know spikes are from one neuron.

Reply:
can you move these plot_raster() comments to the issue #154